chain/network/src/concurrency/scope/mod.rs

```diff
@@ -0,0 +1,444 @@
+//! Asynchronous scope.
+//! WARNING: this module assumes that [panic=abort] is enabled.
```

Comment:
Can we use `catch_unwind` and `std::process::abort` on `Err`? The catch_unwind should be optimized away on panic=abort, and will allow removing this warning.

Reply:
I think we can, I just didn't think it is worth it. I'll try to give it a shot.